script.js

```diff
@@ -0,0 +1,52 @@
+/* Abstraction of game board for bonus features (i.e. checking if player won the game.)
+var board = [[null, null, null],[null, null, null],[null, null, null]];
+console.log(board);
+*/
+
+var turnCounter = 0;
+
+// Select the board, as the parent element of the cells that will be changed, and add event listener
+var gameboard = document.getElementById("board");
+gameboard.addEventListener("click", changeSymbol, false);
+
```

Comment:
the useCapture argument actually defaults to false, so you won't need to explicitly specify it as false. Read [this](http://stackoverflow.com/questions/7398290/unable-to-understand-usecapture-attribute-in-addeventlistener) if you wanna know more about useCapture.